Parser/tokenizer.c

```diff
@@ -618,7 +618,7 @@ static char *
 translate_newlines(const char *s, int exec_input, struct tok_state *tok) {
     int skip_next_lf = 0;
     size_t needed_length = strlen(s) + 2, final_length;
-    char *buf, *current;
+    char *buf, *current, *result;
```

Comment:
Can you move result declaration where it's used? Python 3.8 now allows C99 ;-) (see PEP 7)

Reply:
Done! :)